### DISCUSSION TITLE
Bit of feedbank from a happy user

### DISCUSSION THREAD

**Discussion by RandomBloke85** [immich-app/immich#2630] [General]
Hello

Love the interface, it's clean functionality and beeing able to share albums with family and friends. I'm not the biggest Docker fan, but the installation process couldn't be more easy.

I'm still trying to figure out all the cloud signs in the photo on immich:

cloud not crossed out = photo is in the cloud
cloud crossed out = photo is on the phone but not in the cloud
cloud with a checkmark = photo is on the cloud and the phone

is that about right?

Random thoughts to consider down the road:

1. If a photo is on the phone but not in the cloud (crossed out cloud) one should have the option to long-click it and upload it in the cloud. Just this (or more selected) picture.

2. When selecting a folder to backup with the cloud sign in the upper right corner, one should have the option to select "none".

3. I really miss an option to bulk download every photo in a zip file. The download in the end should have just one folder per user with all photos in it. No subdirectories.

4. It would be great for small communities (like ours) to have kind of a "News"-tab or "immich-feed". There you would be able to see the latest additions of a user. Maybe with a notification?

5. How about a user homepage? Favourites, latest photos and albums.

Thank you for investing your time in developping such a great app.

**Comment by alextran1502** [comment]
Hello, thank you for the feedback. Below are some thoughts on your feedback

> If a photo is on the phone but not in the cloud (crossed out cloud) one should have the option to long-click it and upload it in the cloud. Just this (or more selected) picture.

This is definitely doable and something we considered adding to improve the user experience.

> When selecting a folder to backup with the cloud sign in the upper right corner, one should have the option to select "none".

This is an interesting idea. However, it can cause some conflict with the way the app is designed to show local assets. As local assets are showing based on the album, you choose to backup. So this is a trade-off of selective backup vs blanket backup. I hope that makes sense.

> I really miss an option to bulk download every photo in a zip file. The download in the end should have just one folder per user with all photos in it. No subdirectories.

This is for web upload, correct? For bulk upload, I recommend using the CLI instead of the web because of the resource limitation of the web client.

> How about a user homepage? Favourites, latest photos and albums.

![image](https://github.com/immich-app/immich/assets/27055614/73683fec-49d8-4280-9caa-99306b8bd5a1)

The timeline is already a user home page, correct? You can see the favorite in its own view and the latest photos on the Explore page.

![image](https://github.com/immich-app/immich/assets/27055614/e3bc6c19-e41c-4f14-b90f-b8742d387ae5)

**Reply by RandomBloke85** [reply]
> This is for web upload, correct? For bulk upload, I recommend using the CLI instead of the web because of the resource limitation of the web client.

No, I meant for download. If I do want to make a backup of every photo once in a while to store them somewhere else. Outside of the server. I can copy them from docker, but then I have all those subdirectories.

> The timeline is already a user home page, correct? You can see the favorite in its own view and the latest photos on the Explore page.

Just realized now that "Your Activity" is available in the app but not on the desktop. So I was confused for a moment.

Yes, true, but what I had in mind was sort of a cross-user "Recently Added" where you can see the latest addition to your instance from every user. Kind of an "Instagram lite".  In that regard I meant not only to view my home page, but other users hompages as well (of course, if they permit to do so). Beeing able to snoop around in the latest in-law photos would make immich more lively. I'm aware of the possibilty to share albums. Just a thought however. I like immich and said goodbye to Nextcloud photos already.

**Reply by jrasm91** [reply]
The idea of a "feed" is interesting. Maybe this could work in conjunction with partner sharing and in the future cross-instance sharing. We might start with notifications, but could potentially turn it into a sort of stream/feed in the future.
